tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -206,22 +186,18 @@
     imfdeepcnv = @[IMFDEEPCNV]
     imfshalcnv = @[IMFSHALCNV]
     imp_physics = @[IMP_PHYSICS]
-    ! Yes, really, I mean it. Stop changing the iopt settings too.
-    ! They need to match the RRFS_dev1 parallel.
-    iopt_alb = 2
-    iopt_btr = 1
-    iopt_crs = 1
-    iopt_dveg = 2
-    iopt_frz = 1
-    iopt_inf = 1
-    iopt_rad = 1
-    iopt_run = 1
-    iopt_sfc = 1
-    iopt_snf = 4
-    iopt_stc = 1
+    iopt_dveg = 4
+    iopt_crs  = 2
+    iopt_btr  = 1
+    iopt_run  = 1
+    iopt_sfc  = 3
+    iopt_frz  = 1
+    iopt_inf  = 1
+    iopt_rad  = 3
+    iopt_alb  = 1
+    iopt_snf  = 4
     iopt_tbot = 2
```

Comment:
none of these iopt_* parameters are used in the RUC LSM, with is used in this RRFS configuration, so I'm not sure why they are being changed (or present).